dotnet/Templates/Microsoft.iOS.Templates/ios/fsharp/Main.fs

```diff
@@ -3,10 +3,10 @@
 open UIKit
 
 module Main =
-    // This is the main entry point of the application.
-    // If you want to use a different Application Delegate class from "AppDelegate"
-    // you can specify it here.
     [<EntryPoint>]
-    let main args = 
+    let main args =
+        // This is the main entry point of the application.
```

Comment:
I had to move this comment inside to let the unit test `CreateAndBuildFSharpItemTemplates` insert the generated code at the right place